src/App.vue

```diff
@@ -90,39 +91,38 @@
                 @haveData="haveData = true"
               />
             </v-col>
-            <v-col cols="12" lg="3" style="position: relative">
 
-              <!-- Corrections still uses the unmodified fee summary -->
-              <template v-if="showFeeSummary && isCorrectionView">
-                <aside>
-                  <affix
-                    relative-element-selector=".col-lg-9"
-                    :offset="{ top: 86, bottom: 12 }">
-                    <sbc-fee-summary
-                      :filingData="[...getFilingData]"
-                      :payURL="payApiUrl"
-                    />
-                  </affix>
+            <!-- right side -->
+            <v-col cols="12" lg="3" style="position: relative">
+              <affix v-if="showFeeSummary"
+                relative-element-selector=".col-lg-9"
+                :offset="{ top: 86, bottom: 12 }"
+              >
+                <!-- Corrections still uses the unmodified fee summary -->
+                <aside v-if="isCorrectionView">
+                  <sbc-fee-summary
+                    :filingData="[...getFilingData]"
+                    :payURL="payApiUrl"
+                  />
                 </aside>
-              </template>
-
-              <!-- Alterations uses the new fee summary shared component -->
-              <fee-summary
-                v-else
-                :show-fee-summary="showFeeSummary"
-                :filing-data="getFilingData"
-                :pay-api-url="payApiUrl"
-                :isBusySaving="isBusySaving"
-                :hasConflicts="isConflictingLegalType && hasNewNr"
-                :isSummaryMode="isSummaryMode"
-                @action="handleSummaryActions($event)"
-              />
 
+                <!-- Alterations uses the new fee summary shared component -->
+                <fee-summary v-if="isAlterationView"
```

Comment:
I decided to go with another v-if here in case we support more types later (plus it means you don't have to search for what this is a v-else of).

Reply:
Agree